### DISCUSSION TITLE
Add RTSP support for MediaElement on iOS

### DISCUSSION THREAD

**Discussion by omegazero2310** [CommunityToolkit/Maui#1017] [New Feature Discussions]
Before MediaElement was added to the Community toolkit, I used VLC with MAUI custom control handler for my project. It just did not work so well because no official support from the library for MAUI and I need to play RTSP video live streaming on Android and iOS.
On the announcement, I can see that the new MediaElement supports RTSP video live streaming but only on Android, not iOS because the iOS underlying implement is AVPlayer.
Can we have the same RTSP support on iOS as Android did?

**Comment by mrwcjoughin** [comment]
@omegazero2310 why don't you use HLS streaming? Its supported by Apple (in fact invented by Apple), and is the most commonly used streaming standard - even Netflix uses it...

**Reply by omegazero2310** [reply]
Because our customers still use old webcam hardware and don't want to change to a newer one that supports HLS so our app still needs to support that old protocol 
And besides Microsoft Dev Blog wrote that MediaElement supports RTSP but it missing support for iOS

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
